pkg/operation/botanist/kubeapiserverexposure.go

```diff
@@ -17,6 +17,7 @@ package botanist
 import (
 	"context"
 
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1"
```

Comment:
We typically use an alias here:

```suggestion
	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
```